UM/Decorators.py

```diff
@@ -61,11 +61,6 @@ def call_direct(decorated_function):
 #   excluding builtin functions like __getattr__. It is also expected to match the signature of
 #   those functions.
 def interface(cls):
-    # First, sanity check the interface declaration to make sure it only contains methods
-    invalid_properties = list(filter(lambda i: not i[0].startswith("__") and not inspect.isfunction(i[1]), inspect.getmembers(cls)))
-    if invalid_properties:
-        raise TypeError("Class {0} is declared as interface but includes non-method properties: {1}".format(cls, invalid_properties))
-
```

Comment:
To be honest I can't remember any more.

The decorators in this file perform some type checking related tasks at run time. We now have mypy for type checking. The stuff in here is obsolete now.